lms/views/admin/lti_registration.py

```diff
@@ -0,0 +1,131 @@
+from pyramid.httpexceptions import HTTPFound
+from pyramid.renderers import render_to_response
+from pyramid.view import view_config, view_defaults
+from sqlalchemy.exc import IntegrityError
+
+from lms.security import Permissions
+from lms.services import LTIRegistrationService
+from lms.views.admin import flash_missing_fields
+
+
+@view_defaults(request_method="GET", permission=Permissions.ADMIN)
+class AdminLTIRegistrationViews:
+    def __init__(self, request):
+        self.request = request
+        self.lti_registration_service: LTIRegistrationService = request.find_service(
+            LTIRegistrationService
+        )
+
+    @view_config(
+        route_name="admin.registrations",
+        request_method="GET",
+        renderer="lms:templates/admin/registrations.html.jinja2",
+    )
+    def registrations(self):  # pylint: disable=no-self-use
+        return {}
+
+    @view_config(
+        route_name="admin.registration.new",
+        request_method="GET",
+        renderer="lms:templates/admin/registration.new.html.jinja2",
+    )
+    def registration_new(self):  # pylint: disable=no-self-use
+        return {}
+
+    @view_config(
+        route_name="admin.registration.new",
+        request_method="POST",
+    )
+    def registration_new_post(self):
```

Comment:
```suggestion
    def new_registration_callback(self):
```
Not sure about the name, but I feel like "POST" doesn't give a lot away about what's going on?